commcare_connect/opportunity/models.py

```diff
@@ -348,9 +348,7 @@ class CompletedWork(models.Model):
     @property
     def completed_count(self):
         """Returns the no of completion of this work. Includes duplicate submissions."""
-        visits = self.uservisit_set.filter(status=VisitValidationStatus.approved).values_list(
```

Comment:
I can't tell if I asked this previously, but this property is used for accrual right? So it needs to limit to approved visits?

Reply:
[This document](https://docs.google.com/document/d/1XMjmBJPMOIYoQWUYpHFkY2JS2KluIr-oJ3ffV3288N8/edit#heading=h.6tqrqqcvmf19) has changed the criteria now. Any CompletedWork that is approved will accrue payment irrespective of the individual visits.